requirements.txt

```diff
@@ -1,14 +1,15 @@
 Flask_SQLAlchemy==2.4.4
-numpy==1.20.3
+numpy<=1.22
 scikit-image==0.18.1
 scikit-learn==0.24.0
 opencv-python-headless==4.5.3.56
 scipy==1.6.0
 requests==2.25.1
 SQLAlchemy==1.3.22
+networkx<=3.1
 --find-links https://download.pytorch.org/whl/torch_stable.html
-torch==1.8.1+cu111
-torchvision==0.9.1+cu111
+torch==1.8.1+cpu
```

Comment:
I think this means it only installs the CPU version @jacksonjacobs1 ?

Reply:
oh i understand now, this is the base requirements.txt, so we're assuming no GPU is available? not sure i like that approach since i think many folks would just run the install and not notice that their GPU isn't available

i would think to either:
a) copy and rename requirements to requirements_cpuonly.txt
b) looking at the pytorch page - a lot has changed, and one seems to be able to install pytorch now simply by saying:

pip3 install torch torchvision torchaudio 

this didn't used to be the case, one actually needed to provide the URL

maybe the default requirements.txt should use this approach and assume folks are set up to use the "Default" pytorch install?